docs/GetStarted.mdx

```diff
@@ -366,7 +366,7 @@ if (e.enforce(sub, obj, act) == true) {
 
 <TabItem value="Node.js" label="Node.js">
 
-```js
+```javascript
```

Comment:
What's the difference between `js` and `javascript`?

Reply:
nothing. Was testing something, went through into the commit.